clients/ctl/admin-ui/cypress/e2e/datasets.cy.ts

```diff
@@ -391,12 +391,14 @@ describe("Dataset", () => {
         ).to.eql(connectionString);
       });
 
-      // Two requests should be intercepted. (Cypress waits in LIFO order.)
+      // Two requests should be intercepted.
       cy.wait("@postDataset").then((interception) => {
-        expect(interception.request.body.fides_key).to.eql("generated-2");
-      });
-      cy.wait("@postDataset").then((interception) => {
-        expect(interception.request.body.fides_key).to.eql("generated-1");
+        const generatedKeys = [];
+        generatedKeys.push(interception.request.body.fides_key);
+        cy.wait("@postDataset").then((interception2) => {
```

Comment:
Looks like `wait` can take [an array](https://docs.cypress.io/api/commands/wait#Aliases) which could simplify the nesting.

Reply:
oh dang that's beautiful 💯